src/validate-options.ts

```diff
@@ -0,0 +1,48 @@
+import { HoneycombOptions } from './types';
+import {
+  isClassic,
+  MISSING_API_KEY_ERROR,
+  MISSING_SERVICE_NAME_ERROR,
+} from './util';
+
+export const IGNORED_DATASET_ERROR =
```

Comment:
I went back and forth on whether these should live in `util` or not. 🤷🏽‍♀️

Reply:
Yeah it feels better to me to have them all in the same place and the `validate-options` file feels like the right place 👍🏽